keras/backend/theano_backend.py

```diff
@@ -111,6 +111,13 @@ def dot(x, y):
     return T.dot(x, y)
 
 
+def batch_dot(x, y, axes=None):
+    if axes is None:
+        # behaves like tf.batch_matmul as default
```

Comment:
Shouldnt be `behaves like <some theano function> as default`? Instead of referring to a tf function in theano backend?